src/main/resources/db/migration/V1_3__create_table_melding.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE MELDING
+(
+    id                              SERIAL PRIMARY KEY,
+    uuid                            CHAR(36) NOT NULL UNIQUE,
+    created_at                      TIMESTAMPTZ NOT NULL,
+    innkommende                     BOOLEAN NOT NULL,
+    type                            VARCHAR(20) NOT NULL,
```

Comment:
Er dette noe fra emottak, eller vår egen type-definisjon? I så fall, er `melding_type` bedre? Det gir jo mening når det er under `Melding`-tabellen, jeg bare ser for meg at det kommer en enum for denne verdien etterhvert som vil hete `MeldingType` 🤷🏼‍♂️

Reply:
Den enum'en er med allerede: DialogmeldingType (men skal flytte den til en egen kt-fil).